core/codec/cbor/cbor_token.hpp

```diff
@@ -34,6 +34,8 @@ namespace fc::codec::cbor {
     Type type{Type::INVALID};
     uint64_t extra{};
 
+    // TODO (a.chernyshov) Make explicit
+    // NOLINTNEXTLINE(google-explicit-constructor)
```

Comment:
either NOLINTNEXTLINE ignores it's arguments, or `operator bool()` is a constructor

Reply:
`google-explicit-constructor` also disables conversion operators. 
https://clang.llvm.org/extra/clang-tidy/checks/google-explicit-constructor.html